cms/envs/common.py

```diff
@@ -1085,7 +1085,8 @@
 }
 
 DEFAULT_AUTO_FIELD = 'django.db.models.AutoField'
-DEFAULT_HASHING_ALGORITHM = 'sha1'
+
+DEFAULT_HASHING_ALGORITHM = 'sha256'
```

Comment:
https://github.com/django/django/blob/3.2.21/django/conf/global_settings.py#L450
if don't define it will pick 256 from here but defining will more verbose.